pizza_cutter/des_pizza_cutter/_coadd_slices.py

```diff
@@ -279,7 +282,8 @@ def _build_slice_inputs(
         weights = np.array(weights)
         weights /= np.sum(weights)
 
-    return slices, weights, se_info_to_use
+    possible_slices = slices_to_use
+    return slices, weights, possible_slices, se_info_to_use
```

Comment:
I missed this last time around, but the entry from the `src_info` list from desmeds for a given slice is attached to the slice object at `.source_info`. This we do not need to return `se_info_to_use` or keep a copy of it.

Reply:
might as well; I'm using the flags there

Reply:
Is that a yes or a no to my suggestion?

Reply:
it would require a rewrite; I'm keeping both lists so that I can store info on the ones that didn't pass the cut, so I'm not sure exactly how to do it they way you are saying

Reply:
Ohhh ok. I'll branch from your branch and commit the changes in another PR.